### DISCUSSION TITLE
The `os.chdir` function in Python doesn't seem to work in Shadow.

### DISCUSSION THREAD

**Discussion by higher-order** [shadow/shadow#3330] [Q&A] [answered]
I created a simple test script which shows this error:
```python
import os
print("Test started.")
os.chdir("/home/user/")
print("Test ended.")
```
The stderr file of the Shadow host that runs it:
```
Traceback (most recent call last):
  File "/home/user/test.py", line 3, in <module>
      os.chdir("/home/user/")
OSError: [Errno 38] Function not implemented: '/home/user/'
```
The stdout file:
```
Test started.
```
The script works outside of Shadow and prints:
```
Test started.
Test ended.
```
Is this because Shadow doesn't implement some functionality? Will it be simple to implement it?

**Comment by jtracey** [comment, marked as answer]
This is because the `chdir` syscall isn't implemented yet in Shadow, currently tracked as #2960.

**Reply by higher-order** [reply]
I see. I missed the issue because I searched `os.chdir` rather than just `chdir` within the repo. Anyway, thanks for the fast response.
